src/logger.rs

```diff
@@ -2,12 +2,10 @@ use crate::failure::Error;
 use slog::{Drain, Level, LevelFilter};
 use std::str::FromStr;
 
-pub struct Logger {
-    pub instance: slog::Logger,
-}
+pub struct Logger {}
 
 impl Logger {
-    pub fn new(level: &str) -> Result<Self, Error> {
+    pub fn new(level: &str) -> Result<slog::Logger, Error> {
```

Comment:
[Q]: Couldn't this be just a function instead of implementing a struct (since we don't have methods anymore)?

Reply:
Yes, I think that would be ideal, but I like to have consistency with settings and server modules. I'm not yet sure if this would be a bad practice or not